### PR TITLE
FIX: Style and activators for Add bindings and composites button (ISX-1717)

### PR DESCRIPTION
**Pull request by ritamerkl**
### Description

Changed from using ContextualMenuEvents to GenericMenus as ContextualMenus did not support left click for MacOS (see thread [here](https://unity.slack.com/archives/C3414V4UV/p1650493917016099)). The ContextualMenu ends up calling the GenericMenu anyway. Also updated the style and icon of the button. 

### Changes made

fixed: 
- left click access to button menu 
- style and icon of button

### Checklist

Before review:

- [x] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - FogBugz ticket attached, example `([case %number%](https://issuetracker.unity3d.com/issues/...))`.
    - FogBugz is marked as "Resolved" with *next* release version correctly set.
- [ ] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
    - Performance tests.
    - Integration tests.
- [ ] Docs for new/changed API's.
    - Xmldoc cross references are set correctly.
    - Added explanation how the API works.
    - Usage code examples added.
    - The manual is updated, if needed.

During merge:

- [ ] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed case ISX-1436 (UI TK Input Action Asset Editor - Error deleting Bindings with DeleteKey on Windows).
 - Fixed issue with UI Toolkit based Input Action Editor not restoring it's selected items after Domain Reload.
 - Fixed the [`GetHapticCapabilitiesCommand`](xref:UnityEngine.InputSystem.XR.Haptics.GetHapticCapabilitiesCommand) always failing to execute due to a mismatch in the size in bytes of the payload and the size expected by XR devices. Changed [`HapticCapabilities`](xref:UnityEngine.InputSystem.XR.Haptics.HapticCapabilities) to include all properties returned by the XR input subsystem. This makes Input System match the functionality provided by the [XR](https://docs.unity3d.com/Manual/com.unity.modules.xr.html) module's [`InputDevice.TryGetHapticCapabilities`](https://docs.unity3d.com/ScriptReference/XR.InputDevice.TryGetHapticCapabilities.html) and [`HapticCapabilities`](https://docs.unity3d.com/ScriptReference/XR.HapticCapabilities.html).
+- Fixed icon for adding bindings and composites button and added left button click support.
```
Comment (jamesmcgill) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Needs moved into [Unreleased] now that release has gone out.
Would these be 2 different fixes and each have their own line?

**Review by jamesmcgill — APPROVED** [review]
👍

**Review by ekcoh — APPROVED** [review]
LGTM

**Comment by Pauliusd01** [comment]
Checking right now

**Comment by Pauliusd01** [comment]
https://github.com/Unity-Technologies/InputSystem/assets/54306142/64e38a36-9705-496e-97ff-ca5b6d6fc122

This change seems to remove context menu items from the right click, are we okay with that? Seems less convenient (video of me right-clicking on this branch and later on develop)

**Comment by Pauliusd01** [comment]
I'm also getting this error occasionally when right-clicking, is anything from the error familiar to you?
`Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
Could not resolve type with token 01000016 from typeref (expected class 'UnityEditor.IPlatformIconProvider' in assembly 'UnityEditor, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null') assembly:UnityEditor, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null type:UnityEditor.IPlatformIconProvider member:(null)
UnityEditor.RetainedMode:UpdateSchedulers () (at D:/Gitrepo/unity/Modules/UIElementsEditor/RetainedMode.cs:55)
`

**Comment by ritamerkl** [comment]
> Unity_2023-11-24_14-12-53.mp4 
> This change seems to remove context menu items from the right click, are we okay with that? Seems less convenient (video of me right-clicking on this branch and later on develop)

How is this handled in the old IMGUI editor?

**Comment by Pauliusd01** [comment]
> > Unity_2023-11-24_14-12-53.mp4
> > This change seems to remove context menu items from the right click, are we okay with that? Seems less convenient (video of me right-clicking on this branch and later on develop)
> 
> How is this handled in the old IMGUI editor?

Working on both your PR and Develop if set to IMGUI

**Review by Pauliusd01 — APPROVED** [review]
LGTM, if the context menus when right-clicking the action name are re-added.

**Pull request merged**